cylc/flow/workflow_files.py

```diff
@@ -1659,7 +1659,14 @@ def install_workflow(
     if symlinks_created:
         for src, dst in symlinks_created.items():
             install_log.info(f"Symlink created from {src} to {dst}")
-    rundir.mkdir(exist_ok=True)
+    try:
+        rundir.mkdir(exist_ok=True)
+    except FileExistsError:
+        # This occurs when the file exists but is _not_ a directory.
+        raise WorkflowFilesError(
+            f"Cannot install in {rundir} because that path exists, "
+            "and is not a directory."
```

Comment:
Even simpler?
```suggestion
            f"Cannot install as there is an existing file at {rundir}."
```

Reply:
I believe that it can be _anything_ not a directory. However it's much more likely to be a file than anything else.
Feel free to commit if you prefer.

Reply:
I'm outvoted.